code/modules/mob/living/simple_animal/corpse.dm

```diff
@@ -154,4 +154,18 @@
 	corpsehelmet = /obj/item/clothing/head/bearpelt
 
 /obj/effect/landmark/mobcorpse/russian/ranged
-	corpsehelmet = /obj/item/clothing/head/ushanka
\ No newline at end of file
+	corpsehelmet = /obj/item/clothing/head/ushanka
+
+
+/obj/effect/landmark/mobcorpse/russian/ranged/spetsnaz

```

Comment:
are we planning on using these landmarks in mapping? Since they do spawn with a shitload of gear.